metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/params/params_linux.py

```diff
@@ -251,6 +253,26 @@
 threat_intel_join_parallelism = config['configurations']['metron-enrichment-env']['threat_intel_join_parallelism']
 kafka_writer_parallelism = config['configurations']['metron-enrichment-env']['kafka_writer_parallelism']
 
+# Profiler
+
+metron_profiler_topology = 'profiler'
+profiler_input_topic = config['configurations']['metron-enrichment-env']['enrichment_output_topic']
```

Comment:
Was this intentional?  Wouldn't we want a configurable profiler input topic?

Reply:
So, the profiler's input topic, in metron, is the output of the enrichment topology.  I did it this way to keep them in alignment.  There is currently, to my mind, not a usecase where the profiler does take its data from the output of the enrichment topology.

Reply:
Its a good thing to bring up and discuss @merrimanr .  I agree with @cestella .  Doing it this way makes it less error-prone for the user.  If the user changes the enrichment output topic, the Profiler will get auto-magically updated too.  

I cannot think of a use case where the Profiler does not want the output of Enrichment.

Reply:
That makes sense.  I'm good with it.

Reply:
It probably bears thinking about if we want to do the indexing topology this way too, honestly.